src/App.js

```diff
@@ -67,6 +67,28 @@ const App = () => {
 			return rawNumbers.includes(rawPhone(number));
 		});
 
+	const getContactIdByNumbers = (numbers) => {
+		let id;
+		if (numbers?.length) {
+			numbers.forEach((number) => {
+				const contact = getContactByPhone(number);
+				if (contact) id = contact.contactId;
+			});
+		}
+		return id;
+	};
+
+	const getContactByNumbers = (numbers) => {
+		let contact;
+		if (numbers?.length) {
+			numbers.forEach((number) => {
+				const contactFound = getContactByPhone(number);
+				if (contactFound) contact = contactFound;
```

Comment:
Seems strange to keep assigning the variable for each number. Should it `break` upon finding the contact the first time?

Reply:
yes, it should, thanks